modules/common/networkattachment/networkattachment_test.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2023 Red Hat
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package networkattachment
+
+import (
+	"testing"
+
+	networkv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestCreateNetworksAnnotation(t *testing.T) {
+
+	tests := []struct {
+		name      string
+		networks  []string
+		namespace string
+		want      map[string]string
+	}{
+		{
+			name:      "Single network",
```

Comment:
Should this be perhaps "no network" instead?  Just trying to differentiate from the next test labelled `Single network`.

Reply:
done